### ISSUE TITLE
feature - use with include plugin section

### ISSUE THREAD

**Issue by martk70** [mizunashi-mana/dokuwiki-plugin-mdpage#43]
**This is a feature request.**

1st: thank you for this plugin. Wonderful. Even page menu and section links integrate perfectly.

Specification: 
* The 'core' (to me) include-plugin allows adding only a section of a page in another page. 
* [](https://www.dokuwiki.org/plugin:include)
* I did not not manage to embed a section between the markdown tags in another page. It results in the entire page being included. E.g: {{page>mypage#3rdpar}} or {{section>mypage#3rdpar}}

Motivations: 
* Section inclusion on other pages is important to the way I use the wiki (single sourcing)
* Since github markdown is more of a standard, it helps if people can use this 'normal' markdown (also if pasted from other sources).

Thank for this wonderful plugin. It is very useful also without this feature.

**Comment by mizunashi-mana** [comment]
@martk70 Thank you for this report.

Because I am swamped recently, I will need two months to response.  Of cource, PR is welcome.

**Comment by mizunashi-mana** [comment]
## TL;DR

Sorry, I may be not able to support this feature in mdpage plugin.

You should request this feature to `include` plugin.
I am welcome to work together with teams of other plugins.

## Technical Report

DokuWiki manages page contents as arrays of instructions; `plugin` is one of instructions.
`include` plugin uses caches of instructions and analyze it to search target section at https://github.com/dokufreaks/plugin-include/blob/2018-11-29/helper.php#L624 .

Then, because a markdown content is a instruction, a non-parsed content, `include` plugin cannot analyze any markdown contents.
So, we should add some codes analyzing markdown contents to `include` plugin .
